Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -463,6 +506,7 @@ extension TLSConfiguration {
                                  applicationProtocols: [String] = [],
                                  shutdownTimeout: TimeAmount = .seconds(5),
                                  keyLogCallback: NIOSSLKeyLogCallback? = nil,
+                                 sniCallback: NIOSSLSniCallback? = nil,
```

Comment:
We cannot add arguments to existing methods, defaulted or not, as it's a semver major.